lib/layouts/conversation_list/pinned_conversation_tile.dart

```diff
@@ -0,0 +1,486 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:assorted_layout_widgets/assorted_layout_widgets.dart';
+import 'package:bluebubbles/blocs/chat_bloc.dart';
+import 'package:bluebubbles/helpers/socket_singletons.dart';
+import 'package:bluebubbles/helpers/utils.dart';
+import 'package:bluebubbles/layouts/conversation_list/pinned_tile_text_bubble.dart';
+import 'package:bluebubbles/layouts/conversation_view/conversation_view.dart';
+import 'package:bluebubbles/layouts/widgets/contact_avatar_group_widget.dart';
+import 'package:bluebubbles/layouts/widgets/message_widget/reactions_widget.dart';
+import 'package:bluebubbles/layouts/widgets/message_widget/typing_indicator.dart';
+import 'package:bluebubbles/layouts/widgets/theme_switcher/theme_switcher.dart';
+import 'package:bluebubbles/managers/current_chat.dart';
+import 'package:bluebubbles/managers/new_message_manager.dart';
+import 'package:bluebubbles/managers/settings_manager.dart';
+import 'package:bluebubbles/repository/models/chat.dart';
+import 'package:bluebubbles/repository/models/handle.dart';
+import 'package:bluebubbles/repository/models/message.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:get/get.dart';
+
+class PinnedConversationTile extends StatefulWidget {
+  final Chat chat;
+  final bool? onTapGoToChat;
+  final Function? onTapCallback;
+  final List<File> existingAttachments;
+  final String? existingText;
+
+  PinnedConversationTile({
+    Key? key,
+    required this.chat,
+    this.onTapGoToChat,
+    this.existingAttachments = const [],
+    this.existingText,
+    this.onTapCallback,
+  }) : super(key: key);
+
+  @override
+  _PinnedConversationTileState createState() => _PinnedConversationTileState();
+}
+
+class _PinnedConversationTileState extends State<PinnedConversationTile> with AutomaticKeepAliveClientMixin {
+  bool isFetching = false;
+  Brightness? brightness;
+  Color? previousBackgroundColor;
+  bool gotBrightness = false;
+
+  // Typing indicator
+  RxBool showTypingIndicator = false.obs;
+
+  void loadBrightness() {
```

Comment:
I could be wrong but from what I remember migrating settings this bit of code doesn't actually do anything... We may be able to remove this function entirely.

Reply:
Since that function is in literally every widget, would it be better to remove them all at once in a different pr?

Reply:
Heh probably didn't realize we used it everywhere